src/StatusQ/Components/StatusExpandableSettingsItem.qml

```diff
@@ -0,0 +1,150 @@
+import QtQuick 2.14
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+Rectangle {
+    id: root
+
+    property alias icon: icon
+    property alias primaryText: primaryText
+    property alias secondaryText: secondaryText
+    property alias button: button
+    property alias expandableComponent: expandableRegion.sourceComponent
+
+    property bool isExpandable: true
+    property bool hasButton: false
+    property  bool isExpanded: false
+    property bool hasDropDownArrow: false
+
+    anchors.left: parent.left
+    anchors.leftMargin: 20
+    anchors.right: parent.right
+    anchors.rightMargin: 20
+
+    radius: 8
+    color: Theme.palette.indirectColor1
+    border.color: Theme.palette.baseColor2
+    state: "COLLAPSED"
+    clip: true
+
+    StatusRoundIcon {
+        id: icon
+        anchors.top: parent.top
+        anchors.topMargin: 25
+        anchors.left: parent.left
+        anchors.leftMargin: 11
+
+        icon.background.width: 32
+        icon.background.height: 32
+        icon.background.color: Theme.palette.primaryColor2
+        icon.color: Theme.palette.directColor1
+    }
+
+    StatusBaseText {
+        id: primaryText
+        anchors.top: parent.top
+        anchors.topMargin: 17
+        anchors.left: icon.right
+        anchors.leftMargin: 10
+        anchors.right: parent.right
+        anchors.rightMargin: 10
```

Comment:
I might be wrong, but sticking this to `parent.right` is problematic when the button is visible as well. The text will probably go behind it.

We should make sure that it doesn't bleed into the button in case it exists.

Reply:
no makes sense. I should limit it to the area before the button in case it is visible